xc/xenctrlext_stubs.c

```diff
@@ -195,6 +196,29 @@ CAMLprim value stub_xenctrlext_domain_set_target(value xch,
 	CAMLreturn(Val_unit);
 }
 
+CAMLprim value stub_xenctrlext_domain_vcpu_hotplug(value xch, value domid, value ncpus)
+{
+	CAMLparam2(domid, ncpus);
```

Comment:
We need to include `xch` here too, so use `CAMLparam3`.

Reply:
Thanks, updated